### ISSUE TITLE
showem not tracking PORTAGE_TMPDIR

### ISSUE THREAD

**Issue by FlyingWaffleDev** [sakaki-/showem#2]
I followed the example from here: https://wiki.gentoo.org/wiki/Portage_TMPDIR_on_tmpfs to move my portage build dir into a tmpfs.  Since then showem has not been tracking the PORTAGE_TMPDIR I'm using through a package.env file so that larger ebuilds don't run out of RAM (firefox, libreoffice, etc).  It works just fine for everything that doesn't have the package.env file attached.

Not that big a deal, but I thought you should know.

**Comment by sakaki-** [comment]
Yes, `showem` makes no attempt to apply package-specific environment changes; it just [queries the global context](https://github.com/sakaki-/showem/blob/fd96b15cc178ca2eda30af3c0c33eeb03fc728db/showem#L31) `PORTAGE_TMPDIR` once at the start.

If you have a simple idea to fix this please drop me a PR. I guess it'd be possible to e.g. have `showem` look in `/var/log/emerge.log` first and then loop in the package-specific env files of any active builds, then cycle through the active set?

best, sakaki

**Comment by sakaki-** [comment]
31 Oct 2020: sadly, due to legal obligations arising from a recent change in my 'real world' job, I must announce I am **standing down as maintainer of this project with immediate effect**. For the meantime, I will leave the repo up (for historical interest, and it may be of use to others); however, I plan no further updates, nor will I be accepting / actioning further pull requests or bug reports from this point. Email requests for support will also have to be politely declined, so, **please treat this as an effective EOL notice**.<br><br>For further details, please see my post [here](https://forums.gentoo.org/viewtopic-p-8522963.html#8522963).<br><br>With sincere apologies, sakaki ><
